lizard.py

```diff
@@ -679,15 +679,25 @@ def print_warnings(option, scheme, warnings):
     warn_str = "!!!! Warnings ({0}) !!!!".format(
         ' or '.join("{0} > {1}".format(
             k, val) for k, val in option.thresholds.items()))
-    print("\n" + "=" * len(warn_str) + "\n" + warn_str)
-    print(scheme.function_info_head())
-    for warning in warnings:
+    for _, warning in enumerate(warnings):
```

Comment:
You're absolutely right. Removing...

On Tue, Sep 13, 2016 at 9:04 AM, Olzhas Rakhimov <notifications@github.com>
wrote:

> In lizard.py
> <https://github.com/terryyin/lizard/pull/151#discussion_r78565022>:
>
> > @@ -679,15 +679,25 @@ def print_warnings(option, scheme, warnings):
> >      warn_str = "!!!! Warnings ({0}) !!!!".format(
> >          ' or '.join("{0} > {1}".format(
> >              k, val) for k, val in option.thresholds.items()))
> > -    print("\n" + "=" * len(warn_str) + "\n" + warn_str)
> > -    print(scheme.function_info_head())
> > -    for warning in warnings:
> > +    for _, warning in enumerate(warnings):
>
> enumerate isn't needed anymore, is it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/terryyin/lizard/pull/151/files/091376eb627a2cefe1fc4bc4cf81a2ece066ced5#r78565022>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFK7iUMVkDS1z1p_TzX38xv3TKPGCF57ks5qpq1lgaJpZM4J0DSV>
> .
>